tests/unit/maxmemory.tcl

```diff
@@ -1,7 +1,11 @@
-start_server {tags {"maxmemory" "external:skip"}} {
+start_server {tags {"maxmemory external:skip"}} {
     r config set maxmemory 11mb
     r config set maxmemory-policy allkeys-lru
     set server_pid [s process_id]
+    # Disable copy avoidance
+    r config set min-io-threads-avoid-copy-reply 0
+    r config set min-string-size-avoid-copy-reply 0
+    r config set min-string-size-avoid-copy-reply-threaded 0
 
```

Comment:
I assume we're disabling copy avoidance because of some of the specific calculations done inside this test, so let's call that out in the comment.